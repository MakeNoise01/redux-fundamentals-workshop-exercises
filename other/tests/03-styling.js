import React from 'react'
import {render} from 'enzyme'
const BoxExample = require(`../../src/${process.env.TEST_DIR ||
  'exercises'}/03-styling`).Example

test('Renders three styled boxes', () => {
  expect(render(<BoxExample />)).toMatchSnapshot()
})
