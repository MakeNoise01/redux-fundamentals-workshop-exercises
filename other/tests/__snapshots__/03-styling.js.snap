// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders three styled boxes 1`] = `
<div>
  <div
    class="Box Box--small"
    style="background-color:lightblue"
  >
    I'm in a small box!
  </div>
  <div
    class="Box Box--medium"
    style="background-color:lightgreen"
  >
    I'm in a medium box!
  </div>
  <div
    class="Box Box--large"
    style="background-color:orange"
  >
    I'm in a large box!
  </div>
</div>
`;
